test/math/FloatMatrixTest.ooc

```diff
@@ -7,6 +7,7 @@ FloatMatrixTest: class extends Fixture {
 	matrix := FloatMatrix new (3, 3)
 	nonSquareMatrix := FloatMatrix new (IntSize2D new(2, 3))
 	nullMatrix := FloatMatrix new(0, 0)
+	tolerance := 0.00001f
```

Comment:
- In just about every other test, this variable is called `precision`.
- Use scientific notation: `1.0e-5f´ (makes it easier to count the zeroes).